kuka_kr6_support/urdf/kr6r900_2_macro.xacro

```diff
@@ -1,114 +1,128 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find kuka_resources)/urdf/common_properties.xacro"/>
+  <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
 
-  <xacro:macro name="kuka_kr6r900_2" params="prefix">
-    <xacro:include filename="$(find kuka_resources)/urdf/common_materials.xacro"/>
+  <xacro:macro name="kr6r900_2" params="prefix parent *origin">
```

Comment:
Please revert this and follow what @gavanderhoorn explained. Add top link outside of the macro.